apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -563,13 +563,17 @@ void _wt_scan(void *arg)
 
 	struct wt_options *ap_info = (struct wt_options *)arg;
 	if (ap_info->scan_specific) {
-		wifi_manager_ap_config_s config;
+		wifi_manager_scan_config_s config;
 		memset(&config, 0, sizeof(config));
-		config.ssid_length = strlen(ap_info->ssid);
-		strncpy(config.ssid, ap_info->ssid, config.ssid_length + 1);
-		res = wifi_manager_scan_specific_ap(&config);
+		if (ap_info->ssid) {
+			config.ssid_length = strlen(ap_info->ssid);
+			strncpy(config.ssid, ap_info->ssid, config.ssid_length + 1);
+		} else if (ap_info->channel > 0) {
```

Comment:
Even though channel number is positive, it would be an invalid number, for example, 1000. What happens?

Reply:
I added channel checking  to wifi_manager_scap_ap() in wifi_manager_api.c